app/Http/Controllers/Auth/PasswordController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Validation\Rules\Password;
+
+class PasswordController extends Controller
+{
+    /**
+     * Update the user's password.
+     */
+    public function update(Request $request): RedirectResponse
+    {
+        $validated = $request->validateWithBag('updatePassword', [
```

Comment:
Wellicht nog een check handig of het nieuwe wachtwoord niet hetzelfde is als het oude. Idem bij het wijzigen van het profiel

Reply:
qua veiligheid is het niet handig om een check te doen of het wachtwoord al bestaat, je kan het beter gewoon updaten naar hetzelfde